ext/pcre/php_pcre.c

```diff
@@ -2939,6 +2947,108 @@ static PHP_FUNCTION(preg_last_error)
 }
 /* }}} */
 
+static int callout_func(pcre2_callout_block *block, void *user_data) {
+	const char *subject = (const char *) block->subject;
+	PCRE2_SIZE *ovector = block->offset_vector;
+	zval info, matches, retval;
+	uint32_t i;
+	(void) user_data;
```

Comment:
Should used_data be probably picked up as well? One could pass some user defined object there. Would require preg_callout_function() to be extended to save that. Could be complicated with GC, though.

Thanks.

Reply:
I don't think we need user data arguments anymore nowadays, because you can just `use()` it in the closure instead.

Reply:
Ack, makes sense :) Thanks.